client/src/modules/Score/hooks/types.ts

```diff
@@ -1,8 +1,8 @@
 export type ScoreTableFilters = {
-  githubId?: string;
-  name?: string;
-  'mentor.githubId'?: string;
-  cityName?: string;
+  githubId?: string | string[];
+  name?: string | string[];
+  'mentor.githubId'?: string | string[];
+  cityName?: string | string[];
```

Comment:
could you explain how filter could have those properties as arrays?

Reply:
This is the data we get when we make a filter.

![filter-props](https://github.com/rolling-scopes/rsschool-app/assets/102549906/026041c5-62f7-41af-9f5e-99560bf006ca)